src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Model/ContactPersonEncounter.swift

```diff
@@ -6,13 +6,24 @@ import Foundation
 
 struct ContactPersonEncounter: Equatable {
 
-	// MARK: - Init
+	// MARK: - Types
 
 	enum Duration: Int {
 		case none
 		case lessThan15Minutes
 		case moreThan15Minutes
 
+		var description: String {
+			switch self {
+			case .none:
+				return ""
+			case .lessThan15Minutes:
+				return AppStrings.ContactDiary.PersonEncounter.durationLessThan15Minutes
+			case .moreThan15Minutes:
+				return AppStrings.ContactDiary.PersonEncounter.durationMoreThan15Minutes
+			}
+		}
+
 		var germanDescription: String {
```

Comment:
General: Is it wanted, that the variable name is bound to German `germanDescription`?

Reply:
I thought its OK to name it like that, because it returns the german wording. It is needed for the export, which is in german only at the moment.

Reply:
Just was thinking, that non speaking people in Germany could enter texts in their language.